src/nanotron/trainer.py

```diff
@@ -619,8 +720,9 @@ def train_step_logs(
                 wandb.log(
                     {
                         **{log_item.tag: log_item.scalar_value for log_item in log_entries},
-                        "iteration_step": self.iteration_step,
-                    }
+                        # "iteration_step": self.iteration_step,
```

Comment:
```suggestion
                        "iteration_step": self.iteration_step,
```
I'd say we keep this to be consistent with past logs.